sig/datadog/appsec/contrib/rails/ext.rbs

```diff
@@ -0,0 +1,14 @@
+module Datadog
+  module AppSec
+    module Contrib
+      module Rails
+        # Rack integration constants
+        module Ext
+          APP: "rails"
+
+          ENV_ENABLED: "DD_TRACE_RAILS_ENABLED"
+        end
```

Comment:
Does steep complain if we change these constants (especially given we're not checking this folder), or do we need to remember to keep them up-to-date?

Reply:
I have to check but I think it does. This can also happen on method arguments and return values that have a known constant value or list of values possible.

When I wrote some of these by hand I put the type in instead of the value though. This is one of the things about which I'm not sure yet what would be best. e.g does Steep check for such static values at call sites and for method body returns, kind of like typed C/Go enums? I can see that being useful balanced against the value duplication.